adapters/httpapi_winhttp.c

```diff
@@ -434,6 +434,9 @@ static HTTPAPI_RESULT InitiateWinhttpRequest(HTTP_HANDLE_DATA* handleData, HTTPA
     const wchar_t* requestTypeString;
     size_t requiredCharactersForRelativePath;
     wchar_t* relativePathTemp = NULL;
+    DWORD supported_protocols = WINHTTP_FLAG_SECURE_PROTOCOL_TLS1 | // TLS 1.0 is support for back-compat reasons (https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-security-deployment)
```

Comment:
Could we just go TLS1_2?  The doc referenced suggests using 1.2 and since Windows has this, couldn't we just lock on this?

Reply:
As discussed offline with John, I will keep 1.0 for back compat reasons, given that the SDL instructions do not comdemn it. 

---
In reply to: [331298041](https://github.com/Azure/azure-c-shared-utility/pull/391#discussion_r331298041) [](ancestors = 331298041)